rpc/src/main/scala/org/bitcoins/rpc/client/Client.scala

```diff
@@ -0,0 +1,168 @@
+package org.bitcoins.rpc.client
+
+import java.util.UUID
+
+import akka.http.javadsl.model.headers.HttpCredentials
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model._
+import akka.stream.ActorMaterializer
+import akka.util.ByteString
+import org.bitcoins.core.config.NetworkParameters
+import org.bitcoins.core.crypto.ECPrivateKey
+import org.bitcoins.rpc.serializers.BitcoindJsonWriters._
```

Comment:
This line show's as unused in Intellij (and is removed if I use the optimize imports), but I need the implicits in there to get it to compile. Am I doing something wrong?

Reply:
Nope, you’re just fine, IntelliJ just isn’t smart enough to see some implicit that are called within others

Reply:
Is it possible to prevent Intellij from removing that line on import optimization?